notebooks/en/chain_of_verification.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>This paragraph can be much shorter: </p><p> </p><p> </p><p>The next code cell sets up a 'CoVe prompt template' within a structure known as AIConfig. AIConfig is a data format to organize prompt templates and specific model settings. To create a config you can either:</p><p>* Generate it using the <a href="https://marketplace.visualstudio.com/items?itemName=lastmile-ai.vscode-aiconfig" target="_blank">AIConfig Vscode Extension</a>, or</p><p>* Download the config <a href="https://github.com/lastmile-ai/aiconfig/blob/main/cookbooks/Chain-of-Verification/cove_template_config.json" target="_blank">here</a> and load it with: </p><p><code>config = AIConfigRuntime.load('cove_template_config.json')</code>. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>